### DISCUSSION TITLE
DrawingView, add support to save images in different formats

### DISCUSSION THREAD

**Discussion by zleao** [CommunityToolkit/Maui#1225] [New Feature Discussions]
Currently In the DrawingView control we cannot specify in which format are the images saved.
For iOS and Windows, they are saved as PNG
For Android, as JPEG

In my case, I'm using the DrawingView to collect signatures (white brackground with black lines) and it would make more sense to generate the image as PNG. Also, since this is a migration from XamForms, I would like to keep the same output of image types...

Adding this option should not be very complicated, and wondering what you guys think?

Also, any reason why on Android, the images are saved as JPEG?

Thanks and great job with this toolkit!

**Comment by VladislavAntonyuk** [comment]
#1251
